tests/lcs/core/graphics/graphics.livecodescript

```diff
@@ -90,4 +90,163 @@ TestAssert "test", not ("4,9" is not within "-5,-10,5,10")
 TestAssert "test", "-6,-11" is not within "-5,-10,5,10"
 TestAssert "test", "5,10" is not within "-5,-10,5,10"
 
-end TestGraphicsIsNotWithin
\ No newline at end of file
+end TestGraphicsIsNotWithin
+
+-- new brush mapping for 101..135
```

Comment:
I don't think that's correct. PI_PATTERNS as defined in parsedef.h is 136, so the brushes stop at 135.